features/step_definitions/machine.rb

```diff
@@ -20,12 +20,16 @@
   # In an UPI deployment, if enabled machinesets, machines should be linked to nodes
   machines = BushSlicer::MachineMachineOpenshiftIo.list(user: admin, project: project("openshift-machine-api"))
   if machines.length == 0
-    raise "Machinesets are not enabled, there are no machines"
+    logger.warn "Machinesets are not enabled, there are no machines"
+    logger.warn "We will skip this scenario"
+    skip_this_scenario
   end
 
   machines.each do | machine |
     if machine.node_name.nil?
```

Comment:
This is condition for unstable cluster , we can not skip , in that case.

Reply:
@miyadav According to Zhaohua's comment, we can debug the failure in that specific failed case, not needing to debug all the cases which failed because of other case failed. https://github.com/openshift/verification-tests/pull/3254#issuecomment-1465882362